sql/telemetry/fxa_users_daily_v1/query.sql

```diff
@@ -59,7 +59,8 @@ WITH
     udf_mode_last(ARRAY_AGG(os_name) OVER w1) AS os_name,
     udf_mode_last(ARRAY_AGG(os_version) OVER w1) AS os_version,
     udf_contains_tier1_country(ARRAY_AGG(country) OVER w1) AS seen_in_tier1_country,
-    udf_contains_registration(ARRAY_AGG(event_type) OVER w1) AS registered
+    udf_contains_registration(ARRAY_AGG(event_type) OVER w1) AS registered,
+    COUNTIF(event_type != 'fxa_rp - engage') OVER w1 = 0 AS rp_engage_only
```

Comment:
This new `rp_engage_only` dimension is set to true when the only engagement event we saw from a user in a given day was `fxa_rp - engage`, with the idea that we use this down the line to be able to track MAU unaffected by this event type.

Does that sound reasonable, @irrationalagent ?

Reply:
Yes, this seems like it would work for now. However long term other Relying Parties (RPs) of FxA may want to use this event. As implemented here, it would flag all users generating this event regardless of RP. If we want to selectively flag users who generated the event via monitor, we would have to check for `service = fx-monitor` in addition. I don't know of any other RPs that have plans in the works to this right now, but FxA are setup to support them if they want to.